### PR TITLE
Adding listen & control url parameters

### PR DESCRIPTION
**Pull request by neelimamukiri**

### CODE REVIEW

**Comment by DivyaVavili** [comment]
Are we using contivInit.sh? If so, we have to expose it in that script as well. If not, we have to cleanup the file.

**Comment by DivyaVavili** [comment]
Changes in this file LGTM

**Comment by neelimamukiri** [comment]
We are not using contivInit.sh for the k8s installer. I am not sure if it is being used else where. We can make the changes to that and test it in a different PR - if it is being used else where.

**Comment by neelimamukiri** [comment]
@DivyaVavili, @rhim, we need this PR for the corresponding installer change https://github.com/contiv/install/pull/108

File: scripts/netContain/scripts/contivNet.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@ cleanup=false
 cstore_param=""
 vtep_ip_param=""
 vlan_if_param=""
+control_url=":9999"
```
Comment (rchirakk) [inline scripts/netContain/scripts/contivNet.sh]: do you need to pass default control_url/listen_url ?
Isn't that taken care by netmaster if not specified ?

Comment (neelimamukiri) [inline scripts/netContain/scripts/contivNet.sh]: You don't need to pass the arguments to netmaster. These are the defaults.

**Pull request merged**